dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -479,7 +547,339 @@ private void moveAssetsToBackupDir() throws FileNotFoundException, IOException{
 
 
 	/**
->>>>>>> origin/master
+	 * This method will pull a list of all tables /classed being managed by
+	 * hibernate and export them, one class per file to the backupTempFilePath
+	 * as valid XML. It uses XStream to write the xml out to the files.
+	 *
+	 * @throws ServletException
+	 * @throws IOException
+	 * @author Will
+	 * @throws DotDataException
+	 */
+	private void createXMLFiles() throws ServletException, IOException, DotDataException, DotSecurityException {
+
+//		deleteTempFiles();
+
+		Logger.info(this, "Starting createXMLFiles()");
+
+		Set<Class> _tablesToDump = new HashSet<Class>();
+		try {
+
+			/* get a list of all our tables */
+			Map map = new HashMap();
+			//Including Identifier.class because it is not mapped with Hibernate anymore
+			map.put(Identifier.class, null);
+
+			map.putAll(HibernateUtil.getSession().getSessionFactory().getAllClassMetadata());
+
+			Iterator it = map.entrySet().iterator();
+			while (it.hasNext()) {
+				Map.Entry pairs = (Map.Entry) it.next();
+				Class x = (Class) pairs.getKey();
+				if (!x.equals(Inode.class) && !x.equals(Clickstream.class) && !x.equals(ClickstreamRequest.class) && !x.equals(Clickstream404.class))
+					_tablesToDump.add(x);
+
+			}
+
+			XStream _xstream = null;
+			HibernateUtil _dh = null;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)